Neos.Flow.Log/Classes/PsrLogger.php

```diff
@@ -0,0 +1,114 @@
+<?php
+namespace Neos\Flow\Log;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Log\Backend\BackendInterface;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LoggerTrait;
+use Psr\Log\LogLevel;
+
+/**
+ * PSR-3 supporting logger.
+ *
+ * @api
+ */
+class PsrLogger implements LoggerInterface
```

Comment:
Said it earlier, but here it's even worse. This is not a logger for PSRs, but a PSR-3 compatibe logger.